api/services/contact_fields_mapping_service.py

```diff
@@ -101,7 +101,14 @@ def update_mapped_fields(
         try:
             table_data = table_object.get_by_user_id(user_details.id)
             if table_data:
-                setattr(table_data, column_name, mapped_table_column_value)
-                db.session.commit()
+                for table_data_column, table_data_column_value in vars(
+                    table_data
+                ).items():
```

Comment:
I think we can avoid for loop by just taking the column value from the table_data. For this we can use getattr function of python I guess

Reply:
Updated the condition by using getattr function.